### ISSUE TITLE
Add Basic Excalibur Diagnostics tool

### ISSUE THREAD

**Issue by eonarheim** [excaliburjs/Excalibur#97]
Something like

```
ex.Diagnostics.getFps()
ex.Diagnostics.getActorCount()
ex.Diagnostics.getActorPoolUtilization()
ex.Diagnostics.getLogs()
ex.Diagnostics.getMemoryUsage()
ex.Diagnostics.getTimeline()
ex.Diagnostics.showToolbar() //use the dom to show diagnostics window;

```

Also we should include Excalibur build information in the diagnostics. 

Resources:

- https://docs.webplatform.org/wiki/apis/timing/properties/memory

**Comment by eonarheim** [comment]
Perhaps use `ex.debug` insteead of `ex.Diagnostics` like suggested in #390

**Comment by kamranayub** [comment]
Part of this was done as part of 0.8.0 release, moving to 0.9.0 for more features. It might be better to split this up into separate defined chunks.

**Comment by jedeen** [comment]
I recommend just pulling the features that didn't get added yet into their own new issue, referencing this issue, and throwing that new issue in 0.9, so we can close this for 0.8.

**Comment by kamranayub** [comment]
Agreed

**Comment by jedeen** [comment]
closed via #694
